### PR TITLE
added second params for error handling

### PR DESCRIPTION
**Pull request by sivaramakrishnan-techconative**
response header like a network tab
removed response status and placed the error message in the response body

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -25,10 +26,11 @@ export default function App() {
       authorizationUrl: "", // /projects/{projectID}/oauth2/{providerId}/authorizationUrl
     },
     error: {
-      errorFunction: (msg: string) => {
-        alert(msg)
+      errorFunction: (msg: string, response?: AxiosResponse) => {
+        alert(msg);
+        console.log(response)
```
Comment (vivek-dhayalan) [inline src/App.tsx:31]: Remove this console message if this is no more required.

File: src/App.tsx [hunk 2]
```diff
@@ -25,10 +26,11 @@ export default function App() {
...
       },
-      errorMethod: "default",
+      errorMethod: "customFunction",
```
Comment (vivek-dhayalan) [inline src/App.tsx:33]: Why do you want customFunction should be the default errorMethod?

Comment (sivaramakrishnan-techconative) [reply src/App.tsx:33]: image
customFunction is not the default method. It is one of the methods.

Comment (vivek-dhayalan) [reply src/App.tsx:33]: Okay

**Review by vivek-dhayalan — APPROVED** [review]

**Pull request merged**